src/pages/events/project-setup.md

```diff
@@ -1,58 +1,179 @@
+
+1. Select your project's organization.
+
+1. Select your project.
+
+1. Select the  **DX Experience Cloud SPA** option. The command then initializes a project with a default UI and creates a default Adobe I/O Runtime Action with a name similar to `dx-excshell-1/__secured_generic-<organization-id>-<project-name>-<workspace-name>`. The event configuration process requires that at least one action be defined.
+
+1. You can optionally launch App Builder by running the following command:
+
+   ```bash
+   aio app run
+   ```
+
+## Begin configuring events on Commerce
+
+You must configure Commerce to communicate with your project. You will need the files that you downloaded from the the Adobe Developer Console.
+
+1. In the Commerce Admin, navigate to **Stores** > Settings > **Configuration** > **Adobe Services** > **Adobe I/O Events** > **General configuration**.
+
+1. Copy and paste the contents of the `private.key` file into the **Service Account Private Key** field. Use the following command to copy the contents.
+
+   ```bash
+   cat config/private.key | pbcopy
+   ```
+
+1. Copy the contents of the `<workspace-name>.json` file into the **Adobe I/O Workspace Configuration** field.
+
+1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value can be any string, but it must be unique.
+
+1. Click **Save Config** but do not leave the page.
+
+## Create an event provider and finish Commerce configuration
+
+You cannot create an event provider until after you have configured and saved the private key, workspace file, and instance ID values.
+
+1. Create a `<Commerce-root-directory>/app/etc/event-types.json` file and add the following contents:
+
+   ```json
+   {
+    "provider": {
+        "label": "My Adobe Commerce Events",
+        "description": "Provides out-of-process extensibility for Adobe Commerce"
+        }
+    }
+
+1. Run the following command to create an event provider:
+
+   ```bash
+   bin/magento events:create-event-provider
+   ```
+
+   The command displays a message similar to the the following:
+
+   ```terminal
+   No event provider found, a new event provider will be created
+   A new event provider has been created with ID 63a1f8fe-e911-45a4-9d3f
+   ```
+
+1. Copy the ID returned in the command output into the **Adobe I/O Event Provider ID** field in the Admin.
+
+1. Click **Save Config**.
+
+## Subscribe and register events
+
+You must define which Commerce events to subscribe to, then register them in the project.
+
+Commerce provides two sources for events: observers and plugins. You must specify the source as part of the event name. See [Subscribe to a Commerce event](./commands.md#subscribe-to-a-commerce-event) for details about the syntax of the `events:subscribe` command.
+
+1. Use the `events:subscribe` command to subscribe to Commerce events, as shown in the following examples:
+
+   ```bash
+   bin/magento events:subscribe observer.observer.catalog_product_save_after
```

Comment:
The event name here should be `observer.catalog_product_save_after` (what's written above wouldn't have created an error though if you tried it just based on how we're validating observer events)

Reply:
Fixed. That was a bad copy and paste.